### PR TITLE
Capture immutable fields during view creation through web UI.

### PR DESCRIPTION
**Pull request by adityamukho**
### Scope & Purpose

This PR extends the Views web UI, letting it capture View properties that are immutable once created. These include:
1. `primarySort`
2. `storedValues`
3. `primarySortCompression`
4. `writebufferIdle`
5. `writebufferActive`
6. `writebufferSizeMax`

Prior to this fix, any view created through the web UI will necessarily use **default** values for the above properties, and there is no way to change them later.

- [x] :hankey: Bugfix (requires CHANGELOG entry)
- [ ] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [ ] :book: CHANGELOG entry made

#### Backports:

- [x] No backports required
- [ ] Backports required for: *(Please specify versions and link PRs)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket number:
- [ ] Design document: 

### Testing & Verification

*(Please pick either of the following options)*

- [ ] This change is a trivial rework / code cleanup without any test coverage.
- [x] The behavior in this PR was *manually tested*
- [ ] This change is already covered by existing tests, such as *(please describe tests)*.
- [ ] This PR adds tests that were used to verify all changes:
  - [ ] Added new C++ **Unit tests**
  - [ ] Added new **integration tests** (e.g. in shell_server / shell_server_aql)
  - [ ] Added new **resilience tests** (only if the feature is impacted by failovers)
- [ ] There are tests in an external testing repository:
- [ ] I ensured this code runs with ASan / TSan or other static verification tools

Link to Jenkins PR run: https://jenkins.arangodb.biz:3456/view/PR/job/arangodb-matrix-pr/16861/

### Documentation

> All new features should be accompanied by corresponding documentation. 
> Bugs and features should furthermore be documented in the CHANGELOG so that
> support, end users, and other developers have a concise overview. 

- [ ] Added entry to *Release Notes* 
- [ ] Added a new section in the *Manual* 
- [ ] Added a new section in the *HTTP API* 
- [ ] Added *Swagger examples* for the HTTP API  
- [ ] Updated license information in *LICENSES-OTHER-COMPONENTS.md* for 3rd party libraries

### External contributors / CLA Note 

Please note that for legal reasons we require you to sign the [Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/modalView.js [hunk 1]
```diff
@@ -5,21 +5,29 @@
...
-  var createButtonStub = function (type, title, cb, confirm) {
-    return {
+  var createButtonStub = function (type, title, cb, confirm, style) {
+    const button = {
       type: type,
       title: title,
       callback: cb,
       confirm: confirm
     };
+    if (style !== undefined) {
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/modalView.js:15]:
```suggestion
    if (style) {
```

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/modalView.js [hunk 2]
```diff
@@ -5,21 +5,29 @@
...
   };
 
   var createTextStub = function (type, label, value, info, placeholder, mandatory, joiObj,
-    addDelete, addAdd, maxEntrySize, tags) {
+    addDelete, addAdd, maxEntrySize, tags, style) {
     var obj = {
       type: type,
       label: label
     };
+    if (style !== undefined) {
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/modalView.js:28]:
```suggestion
    if (style) {
```

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/modalView.js [hunk 3]
```diff
@@ -5,21 +5,29 @@
...
+      obj.style = style;
+    }
     if (value !== undefined) {
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/modalView.js:31]:
```suggestion
    if (value) {
```

**Review by hkernbach — APPROVED** [review]
Just one very minor thing. LGTM.

**Comment by Dronplane** [comment]
It is not related to this PR but could we include it here also?
at file 
js/apps/system/_admin/aardvark/APP/react/src/views/analyzers/AnalyzersReactView.jsx:174
there is a typo:
"Show In-built Analyzers" should be "Show Built-in Analyzers" to match naming in our docs https://www.arangodb.com/docs/stable/analyzers.html#built-in-analyzers

**Comment by adityamukho** [comment]
I'm working on a dedicated bugfix PR for analyzers web view. I will include
your suggestion there.

File: js/apps/system/_admin/aardvark/APP/frontend/js/templates/modalTable.ejs [hunk 4]
```diff
@@ -1,75 +1,166 @@
...
+                <% }); %>
+                <td>
+                            <span style="display: inline-flex">
+                                <% if (idx) {%>
+                                <button style="margin-left: 5px;"
+                                        class="graphViewer-icon-button gv_internal_remove_line gv-icon-small delete addDelete">
+                                </button>
+                                <% } %>
+                                <% //<i class="moveRowButton moveRowUp disabled fa fa-arrow-circle-up"></i>
+                                // <i class="moveRowButton moveRowDown disabled fa fa-arrow-circle-down"></i> %>
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/templates/modalTable.ejs:111]:
```suggestion
```

File: js/apps/system/_admin/aardvark/APP/frontend/scss/_modals.scss [hunk 5]
```diff
@@ -519,6 +519,24 @@
...
+    }
+
+    &.disabled {
+      color: rgba(0,0,0,.3);
+
+      &:hover {
+        cursor: default;
+      }
+    }
+  }
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/scss/_modals.scss:539]:
```suggestion
```

**Review by hkernbach — CHANGES_REQUESTED** [review]
Two places where we do have unused code. Please remove. Otherwise LGTM.

**Comment by adityamukho** [comment]
> Two places where we do have unused code. Please remove. Otherwise LGTM.

Done!

**Review by hkernbach — APPROVED** [review]

**Pull request merged**